tests/PHPStan/Analyser/nsrt/get-defined-vars.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace GetDefinedVars;
+
+use function PHPStan\Testing\assertType;
+
+$global = "foo";
+
+assertType('array<string, mixed>', get_defined_vars()); // any variable can exist
+
+function doFoo(int $param) {
+	$local = "foo";
+	assertType('array{param: int, local: \'foo\'}', get_defined_vars());
+	assertType('array{\'param\', \'local\'}', array_keys(get_defined_vars()));
```

Comment:
When escaping single quotes, I always switch to double quoted string to prevent that. 

* easier on the eyes
* easier to copy and paste
* easier to search for the unescaped string in the codebase

```suggestion
	assertType("array{param: int, local: 'foo'}", get_defined_vars());
	assertType("array{'param', 'local'}", array_keys(get_defined_vars()));
```

Reply:
I do that too but I checked how it is done in other parts of PHPStan and found out that escaping seems to be standard here.

Reply:
I also often do the double quote. but I think in general it's very inconsistent :D not sure if such thing exists, but a coding standard rule might be good for this 😊

Reply:
We use https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#symplifyquoteescaperector and it works good